runtime/compiler/env/j9methodServer.cpp

```diff
@@ -2559,3 +2557,19 @@ TR_ResolvedRelocatableJ9JITaaSServerMethod::getAttributesCache(bool isStatic, bo
       }
    return attributesCache;
    }
+
+bool
+TR_ResolvedRelocatableJ9JITaaSServerMethod::validateMethodFieldAttributes(const TR_J9MethodFieldAttributes &attributes, bool isStatic, int32_t cpIndex, bool isStore, bool needAOTValidation)
+   {
+   // Validation should not be done against attributes cached per-resolved method
```

Comment:
Could you please expand the comment a little bit explaining why that is the case?